agent/agent.go

```diff
@@ -523,11 +523,11 @@ func (a *Agent) runPostTaskCommands(ctx context.Context, tc *taskContext) {
 
 	// If task sync was requested for the end of this task, run it now.
 	start = time.Now()
-	// TODO (kim): this should probably have a more generous timeout, but don't
-	// know what would be a sane value.
-	syncCtx, cancel := a.withCallbackTimeout(ctx, tc)
-	defer cancel()
 	if taskSyncCmds := endTaskSyncCommands(tc); taskSyncCmds != nil {
+		// TODO (kim): don't know what would be a sane value for this, so make
```

Comment:
should we file a ticket to either make this configurable or annotate that the error in some way? How will user's know that the host is waiting for this timeout?

Reply:
They won't but presumably they asked for it, they know that it's happening. It also appears in the task logs. I guess we could ask the user to specify a timeout, but I don't think even they would know what is a "sane" time to upload their entire task working directory (I imagine they'll just set a huge generous amount of time to make sure it finishes).
https://jira.mongodb.org/browse/EVG-7828